### PR TITLE
[Lag 2] Allow lacp timing tests to retry limited times until succeeded

### PR DESCRIPTION
**Pull request by yxieca**
- [X ] Test case(new/improvement)

### Approach
How did you do it?
lag_2 test has a pretty high failure rate. Most failures are at the lacp frame timing test. This test is testing if DUT would receive LACP frames at the internal consistent with the rate setting, fast: 1 sec, normal: 30 seconds. Test failures are almost always failed to receive 2 packets at the specified interval.

There are too many reasons that could happen. I propose that as long as we receive a pair of frames fits the expected interval within reasonable number of retries (10), the test should pass.

How did you verify/test it?
Repeated execute the lag_2 test, normally the test would fail with 10 retries. With the change, test executed for 70+ times.

### CODE REVIEW

**Comment by lguohan** [comment]
can we get a 10 samples and pick the medium value, if the medium value falls into the range, then we are good.

**Comment by yxieca** [comment]
Sure, I modified the test to collect N (default N=3) packet intervals and test on the median interval.

File: ansible/roles/test/files/acstests/lag_test.py [hunk 1]
```diff
@@ -95,13 +96,46 @@ def setUp(self):
         '''
         self.dataplane = ptf.dataplane_instance
 
+    def getTimeInterval(self, masked_exp_pkt):
```
Comment (lguohan) [inline ansible/roles/test/files/acstests/lag_test.py]: rename to getMedianInterval to reflect the actual function

File: ansible/roles/test/files/acstests/lag_test.py [hunk 2]
```diff
@@ -95,13 +96,46 @@ def setUp(self):
...
+        last_pkt_time = round(float(last_pkt_time), 2)
+
+        for i in range(0, self.interval_count):
+            (rcv_device, rcv_port, rcv_pkt, curr_pkt_time) = self.dataplane.poll(port_number=self.exp_iface, timeout=self.timeout, exp_pkt=masked_exp_pkt)
+
+            # Check the packet received.
+            self.assertTrue(rcv_pkt != None, "Failed to receive LACP packet\n")
+
+            # Get current packet timing
+            curr_pkt_time = round(float(curr_pkt_time), 2)
```
Comment (lguohan) [inline ansible/roles/test/files/acstests/lag_test.py]: not sure if the round op is really needed or not, but anyway.

**Pull request merged**